test/evergreen.yml

```diff
@@ -1447,6 +1447,15 @@ tasks:
           dependent_task: compile-linux-no-ftruncate
       - func: "unit test"
 
+  # Run the tests that uses suite_random with a random starting seed
+  - name: unit-test-random-seed
+    depends_on:
+    - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "unit test"
+        vars:
+          unit_test_args: -v 2 -R cursor13 join02 join07 schema03 timestamp22
```

Comment:
I think this is fine for now, but we should think about how to keep this list up to date.

Reply:
true, should I create a follow up ticket for this @ddanderson ?

Reply:
@boboeee yes, please create a follow up ticket and link it to this one. thanks!